FirebaseAuthUI/FUIAuth.h

```diff
@@ -190,6 +190,11 @@ __attribute__((deprecated("Instead use authUI:didSignInWithAuthDataResult:error:
  */
 @property(nonatomic, copy, nullable) NSURL *TOSURL;
 
+/** @property autoUpgradeAnonymousUsers
+    @brief Whether to enable auto upgrading of anonymous accounts, defaults to NO.
+ */
+@property(nonatomic, assign, getter=isAutoUpgradeAnonymousUsers) BOOL autoUpgradeAnonymousUsers;
```

Comment:
nit: rename this to `shouldAutoUpgradeAnonymousUsers` (for both getter and setter)

Reply:
Thanks, done.